srai/regionizers/base.py

```diff
@@ -23,3 +23,29 @@ def transform(self, gdf: gpd.GeoDataFrame) -> gpd.GeoDataFrame:
 
         """
         raise NotImplementedError
+
+    def _set_crs(self, gdf: gpd.GeoDataFrame) -> gpd.GeoDataFrame:
+        """
+        Set the CRS to WGS84.
+
+        Args:
+            gdf (gpd.GeoDataFrame): GeoDataFrame to be converted.
+
+        Returns:
+            GeoDataFrame converted to EPSG:4326.
+
+        """
+        return gdf.to_crs(epsg=4326)
```

Comment:
I would get rid of this function as it is a bit confusing (set vs to). I would opt for specifying library-wide constant crs="epsg:4326"

Reply:
I added this constant to `srai/utils/constants.py`. I used it everywhere except tests and example notebooks - if you disagree with that let me know.